admin/formatter/class-post-metabox-formatter.php

```diff
@@ -34,6 +41,20 @@ class WPSEO_Post_Metabox_Formatter implements WPSEO_Metabox_Formatter_Interface
 	public function __construct( $post, array $options, $structure ) {
 		$this->post      = $post;
 		$this->permalink = $structure;
+
+		$this->use_social_templates = $this->use_social_templates();
+	}
+
+	/**
+	 * Determines whether the social templates should be used.
+	 *
+	 * @return bool Whether the social templates should be used.
+	 */
+	public function use_social_templates() {
+		return YoastSEO()->helpers->product->is_premium()
+			&& defined( '\WPSEO_PREMIUM_VERSION' )
+			&& \version_compare( \WPSEO_PREMIUM_VERSION, '16.5-RC0', '>=' )
```

Comment:
We shouldn't use the backslash in non-namespaced files (see the two lines above).